service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/ApproveDraftOrdersService.java

```diff
@@ -262,4 +309,13 @@ private State getStateBasedOnNextHearing(CaseData caseData, ReviewDecision revie
         }
         return currentState;
     }
+
+    private String getOthersNotified(List<Element<Other>> selectedOthers) {
+        return Optional.ofNullable(selectedOthers).map(
+            others -> others.stream()
+                .filter(other -> other.getValue().isRepresented() || other.getValue()
+                    .hasAddressAdded())
+                .map(other -> other.getValue().getName()).collect(Collectors.joining(", "))
+        ).orElse(null);
+    }
```

Comment:
SealedOrderHistoryExtraOthersNotifiedGenerator contains same method, i think Soorya moved this into a more generic class that can be used by all, but maybe fine to leave here for now

Reply:
it is not in master yet. I'll create a PR to cleanup before this epic is released